packages/atlas/src/components/_inputs/Select/Select.tsx

```diff
@@ -102,32 +116,43 @@ export const _Select = <T extends unknown>(
             tabIndex={disabled ? -1 : 0}
             size={size}
           >
-            {(valueLabel ?? '') + (selectedItem?.name || placeholder)}
+            {nodeStart && <NodeContainer ref={nodeLeftRef}>{nodeStart}</NodeContainer>}
+            <ValueContainer leftNodeWidth={nodeLeftBounds.width}>
+              {(valueLabel ?? '') + (selectedItem?.name || placeholder)}
+            </ValueContainer>
```

Comment:
Why do we do measuring here? I know this trick was used for regular inputs so that we can have the icon also trigger the input but here we are not dealing with native inputs anymore right? The icon could probably be placed inside some `::before` element. Please try not using measure here if possible